extensions/src/main/java/com/redhat/lightblue/extensions/sequance/Sequance.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.lightblue.extensions.sequance;
+
+import com.redhat.lightblue.metadata.SimpleField;
+
+/**
+ * TODO: Does SimpleField have all the info required by the backend to manage sequences?
+ *
+ * @author mpatercz
+ *
+ */
+public interface Sequance {
+
+    public void createSequance(SimpleField field);
+
+    public void removeSequance(SimpleField field);
+
+    /**
+     * Generate next value in sequance. This should be handled automatically by the backend on insert.
+     *
+     * @param field
+     * @return
+     */
+    public Long nextInSequance(SimpleField field);
```

Comment:
How would a sequence of Objects look like? I wouldn't add it just to have to cast it to BigInteger/Long.